openrndr-core/src/main/kotlin/org/openrndr/shape/ShapeOperations.kt

```diff
@@ -195,6 +204,9 @@ fun difference(from: ShapeContour, with: Shape): Shape {
     }
 }
 
+/**
+ * Applies a boolean difference operation between two [Shape]s.
+ */
 fun difference(from: Shape, with: Shape): Shape {
```

Comment:
Should `with` be renamed to `subtract` as in the next functions? Also for the previous functions? I think it's clearer than `with`.

Reply:
Out of scope for this PR :(